pkg/state/state.go

```diff
@@ -192,7 +196,7 @@ BEGIN
 	VALUES (
 		schemaname,
 		format('sql_%%s', substr(md5(random()::text), 0, 15)),
```

Comment:
While I currently think adding a search path is sufficient, I find it good from to use schema qualified function calls all over,  e.g. also `pg_catalog.format` and `pg_catalog.count(DISTINCT schema_name)`.